src/cargo/util/lock.rs

```diff
@@ -0,0 +1,92 @@
+use file_lock::filename::{Mode, ParseError};
+use file_lock::filename::Lock as FileLock;
+use file_lock::filename::Error as FileLockError;
+
+use std::path::PathBuf;
+use std::fs;
+use std::thread::sleep_ms;
+
+use util::{Config, CargoError, CargoResult, human, caused_human};
+
+pub use file_lock::filename::Kind as LockKind;
+
+impl From<FileLockError> for Box<CargoError> {
+    fn from(t: FileLockError) -> Self {
+        Box::new(t)
+    }
+}
+
+impl From<ParseError> for Box<CargoError> {
+    fn from(t: ParseError) -> Self {
+        Box::new(t)
+    }
+}
```

Comment:
Could you add these to `src/cargo/util/errors.rs`? That's currently where these conversions and trait implementations are housed.